Enterprise-CMCS/cmcs-eregulations
# EREGCSC-1923 -- Get Statute page user ready

Resolves [EREGCSC-1923](https://jiraent.cms.gov/browse/EREGCSC-1923)

**Description:**

Polished Statutes page to be ready to ship to users.  It will still only include one table that displays all SSA statute items regardless of Title.

**This pull request changes:**

- Adds Statutes link to header and tweaks styles to make it work on all screen widths
- Cleans up any lingering style/copy test issues
- Adds more robust testing with some JS unit tests for URL methods, as well as snapshot tests for header and body table cells

**Steps to manually verify this change:**

1. Visit [experimental deployment](https://za77sd8rmb.execute-api.us-east-1.amazonaws.com/dev868) and make sure the statute page is accessible from every page in the site at multiple screen widths without any errors or issues.
